pkg/golang/build.go

```diff
@@ -79,23 +91,70 @@ func (c Environ) Version() (string, error) {
 	return s[2], nil
 }
 
-// Deps lists all dependencies of the package given by `importPath`.
-func (c Environ) Deps(importPath string) (*ListPackage, error) {
+// Find lists all dependencies of the package given by `importPath`.
+func (c Environ) Find(pattern string) ([]*Package, error) {
```

Comment:
I think you can replace this line :)

```suggestion
func (c Environ) Find(pattern string) ([]*packages.Package, error) {return packages.Load(c.Config, pattern)}
```